plasmapy/formulary/relativity.py

```diff
@@ -302,7 +302,7 @@ def _store_velocity_like_argument(
         Take the velocity-like argument and store it via the setter for
         the corresponding attribute.
         """
-        name = list(speed_like_input.keys())[0]
+        name = next(iter(speed_like_input.keys()))
```

Comment:
Didn't realize that we could do this.  This would be better for performance because it's not necessary to create a whole new `list`.